dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -676,6 +672,11 @@ def mypy_folders(self) -> list[str]:
             or self.full_tests_needed
         ):
             folders_to_check.append("dev")
+        if (
+            self._matching_files(FileGroupForCi.TASK_SDK_FILES, CI_FILE_GROUP_MATCHES, CI_FILE_GROUP_EXCLUDES)
+            or self.full_tests_needed
+        ):
+            folders_to_check.append("task_sdk")
```

Comment:
I think we probably would want to change it to `task-sdk` to match `mypy-folder-sdk`? I just check we have no pre-commit hook with `_` in their ids

```suggestion
            folders_to_check.append("task-sdk")
```

Reply:
This is folder name, so it must be `task_sdk` unless we change the folder name to `task-sdk` which we probably do no want :)

Reply:
But yeah. you are right, it should be changed. But the variable name is misleading, it should not be "folders_to_check" but "mypy_checks_to_run" or smth :)

Reply:
And the property should not be "mypy_folders" - but "mypy_chacks" as well.

Reply:
Fixed in https://github.com/apache/airflow/pull/43185/commits/77c786d56f96ef88eef65ef6c0ba6d58811b8820